src/partials/login-tmpl.hbs

```diff
@@ -0,0 +1,24 @@
+
+<div class="login-section">
+    <div class="login-section_info">
+        <h2 class="login-section-heading">Login</h2>
+        <p class="login-section-desc">Get Access to your Orders. Wishlist and Recommendations</p>
+    </div>
+    <form action="#" method="psot" autocomplete="off" class="login-section_form">
+        <div class="form_group">
+            <input type="email" required="required" name="email" placeholder=" " />
+            <span class="form_group-highlight"></span>
+            <span class="form_group-bar"></span>
+            <label class="label-helper" >Email</label>
```

Comment:
Label for attribute missing for all and it is recommended to have it above input.

Reply:
rewrote it